packages/calcite-components/src/components/date-picker-month-header/date-picker-month-header.tsx

```diff
@@ -323,16 +408,127 @@ export class DatePickerMonthHeader {
     commit?: boolean;
     offset?: number;
   }): void {
-    const { yearInput, activeDate } = this;
+    const { yearInputEl, activeDate } = this;
     const inRangeDate = this.getInRangeDate({ localizedYear, offset });
 
     // if you've supplied a year and it's in range, update active date
     if (inRangeDate) {
-      this.calciteInternalDatePickerSelect.emit(inRangeDate);
+      this.calciteInternalDatePickerMonthHeaderSelect.emit(inRangeDate);
     }
 
     if (commit) {
-      yearInput.value = this.formatCalendarYear((inRangeDate || activeDate).getFullYear());
+      yearInputEl.value = this.formatCalendarYear((inRangeDate || activeDate).getFullYear());
+    }
+  }
+
+  private async setSelectMenuIconOffset(select: HTMLCalciteSelectElement): Promise<void> {
+    const iconEl = select.shadowRoot.querySelector("calcite-icon");
+    await componentOnReady(iconEl);
+    const iconContainer = select.shadowRoot.querySelector(".icon-container");
+    const iconWidth = iconEl.getBoundingClientRect().width;
+    const iconContainerWidth = iconContainer.getBoundingClientRect().width;
+    this.selectMenuIconOffsetWidth = iconContainerWidth + (iconContainerWidth - iconWidth) / 2;
+    this.setSelectMenuWidth(select);
+  }
+
+  private setSelectMenuWidth(select: HTMLCalciteSelectElement): void {
+    const selectEl = select.shadowRoot.querySelector("select");
+    let selectedOptionWidth: number;
+    const fontStyle = getComputedStyle(selectEl).font;
```

Comment:
Can the component's host be used to get the `font` property? The props listed [in the doc](https://github.com/Esri/calcite-design-system/blob/4c2c623d59441be8ddc66866748b7b104810b5ac/packages/calcite-components/src/utils/dom.ts#L125) are inherited.

Reply:
After taking a closer look at this, figured currently font in select is set via CSS variable `--calcite-select-font-size`. And the prop is used in select element rather than `:host`. 

We could add `inherit` [here](https://github.com/Esri/calcite-design-system/blob/dev/packages/calcite-components/src/components/select/select.scss#L83)  & reference the font css prop at `:host`. Don't think it was intentional to reference CSS prop in native element [here](https://github.com/Esri/calcite-design-system/pull/1112/files#diff-6292969ff9b2667a39f133113a074309c3d39badaf733e69a7e14b691af447e4R44).

Thoughts?